openhabian-setup.sh

```diff
@@ -67,7 +67,7 @@ if [[ -n "$UNATTENDED" ]]; then
   bashrc_copy
   vimrc_copy
   firemotd_setup
-  java_zulu
+  java_zulu false
```

Comment:
"false" isn't really intuitive. how about a verbal argument like "force-64bit" or similar

Besides I wonder if it wouldn't make sense to divide the huge function into smaller ones!?

Reply:
Totally agree in both cases, period.